app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
+
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  OpenSCAP.oscap_init
```

Comment:
Looks like [this](https://github.com/OpenSCAP/openscap/blob/a50bbd9481dbc4aa1f3fa83902e665652f39d31c/src/common/oscapxml.c#L103) through [this](https://github.com/OpenSCAP/openscap/blob/a50bbd9481dbc4aa1f3fa83902e665652f39d31c/src/common/oscapxml.c#L103) wrapper. maybe @isimluk can elaborate.  

Anyway @jrafanie is there a way to do this we can live with? (Lazy initialization/platform hook)

I will also make sure that this is actually needed